release/tasks/disk-uploader/README.md

```diff
@@ -21,6 +21,19 @@ When user runs [KubeVirt Tekton Tasks](https://github.com/kubevirt/kubevirt-tekt
 
 ### Usage
 
+Secret of the task run:
+```
+apiVersion: v1
+data:
+    accessKeyId: <ACCESS_KEY_ID>
+    secretKey: <SECRET_KEY>
```

Comment:
This requires still instructions how to get the ACCESS_KEY_ID and SECRET_KEY. E.g. the base64 encoding, and the command line to generate this comfortably. Maybe you can also mention that the instructions from https://tekton.dev/docs/pipelines/auth/#configuring-docker-authentication-for-docker will not work here.

Reply:
Can you please point me to any documentation about the format of the accessKeyId and secretKey ? I think even CNV downstream doc is still missing it. If the task would accept the credentials in the default dockerconfigjson format like shown in quay.io's web ui or in tekton documented, I would agree with you. But this task is expecting a format which seems to be special for kubevirt, and so we should document it.